syntax = "proto3";

service ChordService {
  rpc StoreFile(FileRequest) returns (FileResponse);
  rpc LookupFile(FileRequest) returns (FileResponse);
  rpc TransferFile(FileRequest) returns (FileResponse);
}

message FileRequest {
  string filename = 1;
}

message FileResponse {
  string message = 1;
}

message Empty {}
